components/carousel/index.zh-CN.md

```diff
@@ -19,6 +19,8 @@ subtitle: 走马灯
 | --- | --- | --- | --- | --- |
 | afterChange | 切换面板的回调 | function(current) | 无 |  |  |
 | autoplay | 是否自动切换 | boolean | false |  |  |
+| autoplaySpeed | 轮播间隔时长 | number | 3000 |  |  |
+| speed | 图片切换速度 | number | 500 |  |  |
```

Comment:
- `index.en-US.md` should be updated also.

Reply:
ok,it has been commited just now